extern/storage-sealing/fsm.go

```diff
@@ -300,7 +300,9 @@ func (m *Sealing) plan(events []statemachine.Event, state *SectorInfo) (func(sta
 
 	*/
 
-	m.stats.updateSector(m.minerSectorID(state.SectorNumber), state.State)
+	if err := m.onUpdateSector(context.TODO(), state); err != nil {
+		log.Errorw("update sector stats", "error", err)
```

Comment:
shouldn't we be doing something more than just logging?

Reply:
We could do some retrying, but all the errors we can get here are of the "something is horribly broken" nature, so it probably wouldn't even help

Reply:
Yeah, makes sense.